packages/account/tsconfig.json

```diff
@@ -7,7 +7,7 @@
 		"allowJs": true,
 		"jsxImportSource": "@emotion/react"
 	},
-	"include": ["src", "src/**/*.json"],
+	"include": ["src", "src/**/*.json", "../contract/src/accounts/getPair.ts"],
```

Comment:
Is this a mistake? The package shouldn't reference a file in another package.

Reply:
Removed